src/css/_MobileNav.scss

```
@@ -142,3 +142,26 @@ html.mm-background .BookReader {
 @media (min-width: ($brBreakPointMobile + 1)) {
   .BRtoolbar.responsive { display: block; }
 }
+
+//strange css to get two lines ellises https://stackoverflow.com/questions/5269713/css-ellipsis-on-second-line
+li.table-contents-el{
+  overflow: hidden;
+  text-overflow: ellipsis;
+  -webkit-line-clamp: 2;
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  color: gray;
+  font-weight: lighter;
+  padding-top: 4px;
+  padding-bottom: 4px;
+
+}
+
+li.table-contents-el.chapter-clickable{
+  font-weight: normal;
+  color: black;
+}
+
+li.table-contents-el.current-chapter{
+  background-color: lightblue;
+}
```

Since this is Sass, we can actually nest these like so:

```suggestion
  &.chapter-clickable {
    font-weight: normal;
    color: black;
  }

  &.current-chapter{
    background-color: lightblue;
  }
}
```

---

done